pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -138,6 +138,8 @@ enum ServerError {
 
     ProducerBusy         = 16; // Producer with same name is already connected
     InvalidTopicName = 17; // The topic name is not valid
+
+    IncompatibleSchema = 18; // Specified schema was incompatible with topic schema
```

Comment:
do you need to bump the api version? if an old client talks to new brokers, what is the behavior after this change? shall we just respond with BrokerServiceException?

Reply:
Since it's an unknown enum value, it will get defaulted to "UnknownError"